packages/zoe/test/unitTests/contracts/loan/test-borrow.js

```diff
@@ -347,7 +347,7 @@ test('getDebtNotifier with interest', async t => {
   assertAmountsEqual(
     t,
     debtCompounded2,
-    AmountMath.make(40040n, loanKit.brand),
+    AmountMath.make(40041n, loanKit.brand),
```

Comment:
As I understand it, these are increased because now we are getting the ceiling instead of e.g., rounding or getting the floor.

Reply:
Yes, updateDebt sometimes now rounds up from something small to add interest.